src/main/java/backend/IssueMetadata.java

```diff
@@ -12,18 +12,26 @@
     private final List<TurboIssueEvent> events;
     private final List<Comment> comments;
     private final LocalDateTime nonSelfUpdatedAt;
+    private final LocalDateTime selfUpdatedAt;
     private final int nonSelfCommentCount;
+    private final int selfCommentCount;
 
     // If isUpdated is true, nonSelfUpdatedAt will be used to sort/filter instead of updatedAt in TurboIssue
     private final boolean isUpdated;
+    private final boolean isUpdatedBySelf;
+    private final boolean isUpdatedByOthers;
```

Comment:
Yup, I included a difference. The previous implementation of the updated filter was causing some updates to go missed. There's the issue of cache in doing the filtering with `nonSelfUpdatedAt` after metadata is available versus always doing with `updated` whether or not `metadata` is available. The issue has been addressed in #912